ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml

```diff
@@ -8,6 +8,7 @@ ref:
       memory: 100Mi
   env:
     - name: OS_CLOUD
+      default: "openstack"
```

Comment:
This allows us to dictate a default value for OS_CLOUD while maintaining the steps generalization